src/main/java/org/openbase/bco/bcozy/util/ExceptionHelper.java

```diff
@@ -0,0 +1,24 @@
+package org.openbase.bco.bcozy.util;
+
+/**
+ * @author vdasilva
+ */
+public class ExceptionHelper {
```

Comment:
Please refactor to ExceptionProcessor and move it to jul.exception and create there a pull request.
UI independent utils should always be moved to jul.